cypress/integration/userCanViewTimeHistory.feature.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="Cypress" />
+
+describe("User can view history of time recordings", () => {
+	it("successfully displays a list", () => {
+		cy.server();
+		cy.route({
+			method: 'GET',
+			url: 'https://demo-stable.kimai.org/api/timesheets',
+			response: 'fixtures:save_data.json',
+			headers: {
+				"X-AUTH-USER": "susan_super",
+				"X-AUTH-TOKEN": "api_kitten"
+			}
+		})
```

Comment:
Why are you using this response?

Reply:
created a new get_data.json file with that response